src/otx/core/types/export.py

```diff
@@ -112,6 +113,7 @@ def to_metadata(self) -> dict[tuple[str, str], str]:
             ("model_info", "labels"): all_labels.strip(),
             ("model_info", "label_ids"): all_label_ids.strip(),
             ("model_info", "optimization_config"): json.dumps(self.optimization_config),
+            ("model_info", "otx_version"): otx.__version__,
```

Comment:
It looks like we have a merge artifacts in this file: `all_label_ids` can\t be equal to `all_labels` anymore, since we have IDs in `LabelInfo` now.

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4159/commits/2c733c0b005f7896d34f47ba41073049bde134bb